src/askem_beaker/contexts/decapodes/procedures/julia/model_to_equation.jl

```diff
@@ -0,0 +1,11 @@
+using DiagrammaticEquations
```

Comment:
I believe all of these new imports can be throw in into the `setup.jl` for this context rather then being repeated
I may be missing a reason you want to be doing this for this though